test/query.test.js

```diff
@@ -4314,42 +4314,6 @@ describe('Query', function() {
     assert.strictEqual(found[0].title, 'burrito bowl');
   });
 
-  it('update operation should remove fields set to undefined (gh-12794) (gh-12821)', async function() {
```

Comment:
maybe instead of removing this test, convert it to a test to check that `undefined` is actually included?